lib/migrator/index.js

```diff
@@ -135,8 +135,14 @@ const migrate = async () => {
 
     await Promise.all(directoriesToDeleteIfEmpty.map(async (dirPath) => {
       const reporter = await addReporter(`Remove empty directory ${dirPath}`)
-      const result = await deleteDirectoryIfEmpty(dirPath)
-      await reporter(result)
+      // Only report if a directory is empty
+      try {
+        if (await deleteDirectoryIfEmpty(dirPath)) {
+          await reporter(true)
+        }
+      } catch (e) {
+        await reporter(false)
+      }
```

Comment:
We should log the exception, but it is probably easier to just let it propagate to a higher level where we already catch and log errors.


```suggestion
      if (await deleteDirectoryIfEmpty(dirPath)) {
        await reporter(true)
      }
```

Reply:
We might also want to write something to the log file to show we haven't deleted the folder